support-services/src/main/scala/com/gu/support/redemption/CodeValidationResult.scala

```diff
@@ -0,0 +1,30 @@
+package com.gu.support.redemption
+
+import com.gu.support.redemption.corporate.CorporateCodeValidator.CorporateId
+
+sealed abstract class CodeValidationResult(val clientCode: String)
```

Comment:
can we call this CodeStatus as it's a bit more specific than Result and I was mixing it up with RedemptionValidationResult (or not sure if that should also/instead consider a rename?)